### DISCUSSION TITLE
Is there any way to decrease html css size?

### DISCUSSION THREAD

**Discussion by BekaArabidze98** [GrapesJS/grapesjs#5458] [Q&A]
![image](https://github.com/GrapesJS/grapesjs/assets/122085038/19a3ce17-439a-470c-b86a-617c4a86699c)

I send some html css to backend for my project which is generated by grapesjs.
I wanted to ask if there is any way to further decrease the size(sizes on the photo).
Because sometimes page takes too long to load.
I have already taken measures to decrease pictures sizes.

Thanks for the help.

**Comment by nguyenvoanhkhoa1** [comment]
If there is no config for the assets manager, the uploaded image will be embedded as base64.
You need to upload the image to your upload endpoint and then get the image URL.
Read this [doc](https://grapesjs.com/docs/modules/Assets.html#uploading-assets)

**Comment by BekaArabidze98** [comment]
Thanks for the answers. I know that part. I already did that part.
I more like to know If there is any way to minify or decrease size of  html/css
